man/systemd.network.xml

```diff
@@ -4624,6 +4624,15 @@ ServerAddress=192.168.0.1/24</programlisting>
           <xi:include href="version-info.xml" xpointer="v234"/>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Locked=</varname></term>
```

Comment:
commit message: Please use the full hash of the kernel commit, and also mention that is in v5.18 .